tests/torch/ptq/test_fq_lora.py

```diff
@@ -220,3 +226,32 @@ def test_invalid_lora_rank():
             compression_format=CompressionFormat.FQ_LORA,
             advanced_parameters=AdvancedCompressionParameters(lora_adapter_rank=too_big_rank),
         )
+
+
+@pytest.mark.parametrize("all_layers", [True, False])
+def test_compress_shared_weights(mocker, all_layers):
+    model = ShortTransformer(8, 16, share_weights=True)
+
+    input_ids = torch.randint(0, 10, (8,))
+    wrapped_model = wrap_model(model, example_input=input_ids, trace_parameters=True)
+
+    compressed_model = compress_weights(
+        wrapped_model,
+        mode=CompressWeightsMode.INT4_SYM,
+        all_layers=all_layers,
+        group_size=4,
+        compression_format=CompressionFormat.FQ_LORA,
+        advanced_parameters=AdvancedCompressionParameters(lora_adapter_rank=4),
+    )
+    nncf_graph = compressed_model.nncf.get_graph()
+    filename = f"{model.__class__.__name__}_all_layers_{str(all_layers)}.dot"
```

Comment:
```suggestion
    filename = f"compress_shared_weights_all_layers_{all_layers}.dot"
```
ShortTransformer can be used for for new tests.
No need to use str inside f-strings

Reply:
Corrected